portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,153 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Adhya Nagpal</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 id="name">Adhya Nagpal</h1>
+      <br>
+      <img src="../public/AdhyaNagpal.png" width="300" height="300">
+      <!-- <p>Click here to get a random message:</p>
+      <button onclick="addRandomGreeting()">Hello</button> -->
```

Comment:
What is the reason to comment these 2 lines? If it was just for testing you can uncomment them. If you no longer need them then it's better to delete those.

Reply:
These 2 lines were there in the original code for testing purposes. I'll remove them. Thanks.